src/atoms/Flex/__tests__/Flex.web.test.js

```
@@ -7,6 +7,9 @@ import Text from '../../Text';
 import Space from '../../Space';
 import Flex from '../index';
 
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
```

why do you need this?

---

https://github.com/razorpay/blade/pull/81#discussion_r402307800

---

To suppress the error message that comes in test case where we pass more than on child and try to throw error.

---

Will move it to that test case instead of beforeAll

---

Yes please move it to that particular test case